tests/unittests/Foundation/NSProgressTests.mm

```diff
@@ -259,6 +277,7 @@ - (void)addParentTo:(NSProgress*)child {
 }
 
 ARM_DISABLED_TEST(NSProgress, SingleParentEvenAcrossThreads) {
+    #if TARGET_OS_WIN32
```

Comment:
If it is an exception test that isn't checking the result, please use EXPECT ANY THROW. That works on ARM.